routes/swagger.yaml

```diff
@@ -218,20 +218,25 @@ paths:
             application/json:
               schema:
                 $ref: '#components/bodies/definitionList'
-
-  /definitions/{type}/{provider}/{namespace}/{name}:
     get:
```

Comment:
There are other updates to do here as we changed the PATCH /definition contract to the stuff that is written on your whiteboard. I can help you through that

Reply:
I was unsure about this comment and forgot to ask about it today